plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/subscriptions-expired-notice.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import Notice from '~/marketplace/components/notice/notice';
+import { getAdminSetting } from '~/utils/admin-settings';
+
+export default function SubscriptionsExpiredNotice(): JSX.Element | null {
+	const wccomSettings = getAdminSetting( 'wccomHelper', {} );
+	const notice = wccomSettings?.subscription_expired_notice || {};
+
+	if ( ! wccomSettings.isConnected || ! notice?.description ) {
+		return null;
+	}
+
+	const handleClose = () => {
+		const data = { notice_id: 'woo-subscription-expired-notice' };
+		apiFetch( {
+			path: `/wc-admin/notice/dismiss`,
+			method: 'POST',
+			data,
+		} );
```

Comment:
prone to csrf due to missing nonce

Reply:
you are correct, in the API handler we also don't check for nonce, will add that

Reply:
this is addressed in the newer commits